### ISSUE TITLE
Changing user to OOO does not add tags to tickets

### ISSUE THREAD

**Issue by Jamman** [zendesklabs/out_of_office_app#238]
I have tested this on two Zendesk instances.
I am setting an agent to OOO, and NOT selecting the option to unassign all open tickets.
The tickets do not get updated with the OOO tag, and a reply to the ticket is not unassigning the ticket.

**Comment by Jamman** [comment]
Further to this - I  do not seem to be able to manually add the agent_ooo tag either!

**Comment by Jamman** [comment]
OKay, further testing for this - the problem is that only Pending or On Hold tickets receive the OOO tag.
This is not greatly helpful, because if a SOLVED ticket it reopened by reply... it will not reassign to the open group... and if a customer follows up a OPEN ticket, it will not be assigned to the group.

**Comment by Maxcraft90** [comment]
why would you actually not force the unassignment in first place anyway?

**Comment by jondcoleman** [comment]
I agree with @Jamman's comments and think that the tag should be applied to Open tickets, even if that option isn't selected to force un-assignment. Although it appears that the tag is assigned to Solved status tickets.

**Comment by klawrenceHA** [comment]
I'm having the same issue.  There doesn't seem to be any consistency when the Unavailable is selected.  I thought that this would add the ooo tags when it's turned on.  Has there been any updates on this elsewhere?
